docs/cli/edgedb_instance/edgedb_instance_reset_password.rst

```diff
@@ -18,6 +18,11 @@ Description
 ``edgedb instance reset-password`` is a terminal command for resetting
 or updating the password for a user of an EdgeDB instance.
 
+.. note::
+
+    The ``edgedb instance reset-password`` command is not intended for use with
```

Comment:
Actually, maybe this uses the `--admin` flag to connect. It could still be used with "remote" instances but you can't do it remotely. You would have to ssh into the server running the instance. This is speculation on my part at the moment.

Reply:
I've left this one as-is for now.